### PR TITLE
Fix compatibility with Kubernetes v1.18.x

### PR DESCRIPTION
**Pull request by sepetrov**
# Added
- `minikube_kafka` test matrix for Amazon EKS supported Kubernetes versions: v1.14.9, v1.15.11, v1.16.13, v1.17.9, v1.18.9
- `script/build-manifests.sh` to prepare build manifests

# Changed
- `github.com/kubeless/kubeless` from v1.0.0 to v1.0.7
- `github.com/golang/protobuf` from v1.4.2 to v1.2.0 to resolve issue with indirect dependencies causing `go mod tidy` to fail
- `k8s.io/client-go` from v5.0.0 to v7.0.0
- `minikube` from v1.2.0 to v1.13.1

**TODOs**:
 - [x] Ready to review
 - [x] Automated Tests
 - [ ] Docs

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -57,6 +57,11 @@ workflows:
           <<: *build_allways
           requires:
             - build
+          matrix:
+            parameters:
+              # Amazon EKS Kubernetes versions
+              # See https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
+              k8s-version: ["v1.14.9", "v1.15.11", "v1.16.13", "v1.17.9", "v1.18.9"]
```
Comment (andresmgot) [inline .circleci/config.yml:64]: I'd probably left out 1.14 from the list but I'm fine leaving it here as far as it's not troublesome

File: .circleci/config.yml [hunk 2]
```diff
@@ -121,13 +121,19 @@ jobs:
...
     steps:
       - <<: *should_test
       - checkout
       - <<: *exports
+      - run: |
+          echo "export MINIKUBE_KUBERNETES_VERSION=<< parameters.k8s-version >>" >> $BASH_ENV
       - attach_workspace:
           at: /tmp/go
       - <<: *restore_workspace
+      - run: sudo apt-get update && sudo apt-get install -y conntrack
```
Comment (andresmgot) [inline .circleci/config.yml:136]: what's the purpose of `conntrack`?

File: Makefile [hunk 3]
```diff
@@ -82,3 +82,7 @@ bootstrap: bats ksonnet-lib
 
 build_and_test:
 	./script/start-test-environment.sh "make binary && make controller-image CONTROLLER_IMAGE=bitnami/kubeless-controller-manager:latest && make integration-tests"
+
+.PHONY: build-manifests
+build-manifests:
+	./script/build-manifests.sh
```
Comment (andresmgot) [inline Makefile]: missing blank line at the end here

**Review by andresmgot — APPROVED** [review]
Great work again @sepetrov! I just have minor questions/comments.

File: Makefile (hunk 3 above)
Comment (sepetrov) [reply Makefile]: The job for [minikube_kafka-v1.18.9](https://app.circleci.com/pipelines/github/kubeless/kafka-trigger/49/workflows/90744719-a105-4f1d-a23b-923285a934be/jobs/306)  fails if `conntrack` is not installed.

> Kubernetes 1.18.9 requires conntrack to be installed in root's path

Not sure why is required though.

Comment (sepetrov) [reply Makefile]: Fixed

File: .circleci/config.yml (hunk 2 above)
Comment (sepetrov) [reply .circleci/config.yml:136]: The job for [minikube_kafka-v1.18.9](https://app.circleci.com/pipelines/github/kubeless/kafka-trigger/49/workflows/90744719-a105-4f1d-a23b-923285a934be/jobs/306)  fails if `conntrack` is not installed.

> Kubernetes 1.18.9 requires conntrack to be installed in root's path

Not sure why is required though.

File: .circleci/config.yml (hunk 1 above)
Comment (sepetrov) [reply .circleci/config.yml:64]: 👍 

Providing compatibility for as much of the popular (still used) versions will only benefit the project.

We should probably align the integration tests for all repos with the [docs](https://github.com/kubeless/kubeless#compatibility-matrix-with-kubernetes)?

Comment (andresmgot) [reply .circleci/config.yml:64]: Agree (but it's been a long time since we updated that piece of docs)

**Review by andresmgot — APPROVED** [review]

**Pull request merged**